### DISCUSSION TITLE
Is it possible to display an image instead of the liveview and activate it only when an action is launched?

### DISCUSSION THREAD

**Discussion by cedric-Swiss** [photobooth-app/photobooth-app#365] [General]
Hello everyone,

Is it possible to activate an image for the basic screen of the application, instead of the liveview, and only activate the liveview, when an action (Photo or collage) is triggered?

Thank you in advance for your help.

**Comment by mgineer85** [comment]
Hi, this is currently not possible. Maybe it could be done with some css, but I am not sure if it works well.
Feel free to send a feature request so it can be added in the future.

**Reply by cedric-Swiss** [reply]
thanks for your reply. 

too bad
